frontend/src/components/modals/editAccount/useAccountEdit.ts

```diff
@@ -0,0 +1,85 @@
+import { editUsername, updateProfile } from "store/slices/UserSlice";
+import { useAppDispatch, useAppSelector } from "store/store";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { loginSchema } from "components/Form/useForm";
+import { useMutation } from "@tanstack/react-query";
+import axios from "axios";
+import { FormEvent, useState } from "react";
+
+interface MutationResponse {
```

Comment:
Don't name it like this, cause u can have a lot interfaces like this, you must spectify this,

The name should depend of action.

Remeber to use "I" prefix for interfaces and "T" for types.

Here it can be "IUserUpdateResponse"